_includes/blockquote-section.html

```diff
@@ -0,0 +1,15 @@
+<section class="c-blockquote">
+  <div class="c-blockquote__inner-container">
+    <div class="c-blockquote__image-container">
+      <div class="c-blockquote__gradient-container">
+    <img src="assets/GLINTstudios_PSA-Airlines_Dayton_0099.jpg" class="c-blockquote__image" alt="A group photo of the Scout Airlines team">
```

Comment:
This is picky 😜 but could we adjust the indentation in this file so this code is easier to read? For example in this section it would look more like:
```
<div>
  <div>
    <img>
  </div>
</div>
```